cypress/e2e/home-spec.cy.js

```diff
@@ -0,0 +1,46 @@
+describe('rancid tomatillos homepage user flows', () => {
+
+  beforeEach(() => {
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+      statusCode: 200,
+      fixture: '.././fixtures/all-movies-test.json'
+    })
+    .visit('http://localhost:3000');
+  });
+  
+  it('should display all navbar elements upon navigating to the app', () => {
+    cy.contains('h1', 'RANCID')
+    cy.contains('h2', 'TOMATILLOS')
+    cy.get('img[name=tomato-image]')
+      .should('be.visible')
```

Comment:
Curious: if we're already getting the element by using cy.get, would it not be visible already?

Reply:
You're so right! I corrected 👍